video/out/opengl/video.c

```diff
@@ -1183,7 +1183,7 @@ static void dispatch_compute(struct gl_video *p, int w, int h, int bw, int bh)
         // Clamp the texture coordinates to prevent sampling out-of-bounds in
         // threads that exceed the requested width/height
         PRELUDE("#define texmap%d(id) min(texcoord%d_rot(id), vec2(1.0))\n", n, n);
-        PRELUDE("const vec2 texcoord%d = texmap%d(gl_GlobalInvocationID);\n", n, n);
+        PRELUDE("#define texcoord%d texmap%d(gl_GlobalInvocationID)\n", n, n);
```

Comment:
mesa complains:

     error: initializer of const variable `texcoord0' must be a constant expression